homeassistant/components/axis.py

```diff
@@ -59,6 +60,19 @@
     }),
 }, extra=vol.ALLOW_EXTRA)
 
+SERVICE_VAPIX_CALL = 'vapix_call'
+SERVICE_VAPIX_CALL_RESPONSE = 'vapix_call_response'
+SERVICE_CGI = 'cgi'
+SERVICE_ACTION = 'action'
+SERVICE_PARAM = 'param'
+
+SERVICE_SCHEMA = vol.Schema({
+    vol.Required(CONF_NAME): cv.string,
+    vol.Required(SERVICE_PARAM): cv.string,
+    vol.Optional(SERVICE_CGI, default='param.cgi'): cv.string,
+    vol.Optional(SERVICE_ACTION, default='update'): cv.string,
+})
```

Comment:
Use here only `ATTR_*` constante. For Default value also like `DEFAULT_CGI = "param.cgi"` and after that `..., default=DEFAULT_CGI): ...`